github-activity-tracker/.env.example

```
@@ -10,6 +10,10 @@ RDS_DATABASE=github_tracker
 RDS_USER=postgres
 RDS_PASSWORD=your-password
 
+# Single source of truth for lookup tables (seeded into DB on migrate/start).
+GITHUB_ORG=mosip,inji
+USER_ROLES=Developer,Tech Lead,Architect,Product Owner,Leadership,QA Engineer,DevOps Engineer
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)